lib/.eslint-ce-lib.yml

```diff
@@ -17,17 +18,17 @@ rules:
       objects: always-multiline
       imports: always-multiline
       exports: always-multiline
-      functions: never
+      functions: always-multiline
   eol-last:
     - error
     - always
   eqeqeq:
     - error
     - smart
   indent:
-    - error
-    - 4
-    - SwitchCase: 1
+    - off
```

Comment:
There's an edge case or two eslint and prettier disagree on for indentation. Two files in lib/ are affected despite specifying the prettier eslint plugin. Just turning indent off for now.